python/cudf/cudf/core/column/categorical.py

```diff
@@ -141,6 +141,13 @@ def as_ordered(self, inplace: bool = False) -> Optional[SeriesOrIndex]:
             or return a copy of this categorical with
             added categories.
 
+            .. deprecated:: 23.02
```

Comment:
I see `23.02` here but `23.04` below, is this as intended?

Reply:
Yup, this is intentional. `inplace` here was deprecated in `23.02`. This is just updating the docstring.